config/app.php

```diff
@@ -4,6 +4,58 @@
 
 return [
 
+    /*
+    |--------------------------------------------------------------------------
+    | Application Name
+    |--------------------------------------------------------------------------
+    |
+    | This value is the name of your application. This value is used when the
+    | framework needs to place the application's name in a notification or
+    | any other location as required by the application or its packages.
+    |
+    */
+
+    'name' => env('APP_NAME', wp_get_theme()->get('Name')),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Application Environment
+    |--------------------------------------------------------------------------
+    |
+    | This value determines the "environment" your application is currently
+    | running in. This may determine how you prefer to configure various
+    | services your application utilizes. Set this in your ".env" file.
+    |
+    */
+
+    'env' => env('WP_ENV', 'production'),
```

Comment:
This doesn't account for people defining `WP_ENV`, such as if someone has put `define('WP_ENV', 'staging')` in wp-config.php.

It should probably be
```
 => defined('WP_ENV') ? WP_ENV : env('WP_ENV', 'production'),
```